helix-term/src/commands.rs

```diff
@@ -1062,6 +1090,20 @@ mod cmd {
             fun: format,
             completer: None,
         },
+        Command {
+            name: "earlier",
+            alias: None,
```

Comment:
Maybe we can follow vim here.
```suggestion
            alias: Some("ear"),
```